test/focus.spec.js

```diff
@@ -44,4 +44,19 @@ describe('Page.focus', function() {
     expect(focused).toBe(true);
     expect(blurred).toBe(true);
   });
+  it.fail(WEBKIT && !LINUX)('should traverse focus', async function({page, server}) {
+    await page.setContent(`<input id="i1"><input id="i2">`);
+    let focused = false;
+    await page.exposeFunction('focusEvent', () => focused = true);
+    await page.evaluate(() => i2.addEventListener('focus', focusEvent));
```

Comment:
How does this work if nobody awaits the call to focusEvent?

Reply:
I await press